d2graph/d2graph.go

```diff
@@ -397,7 +397,10 @@ func (obj *Object) GetFill(theme *d2themes.Theme) string {
 
 func (obj *Object) GetStroke(theme *d2themes.Theme, dashGapSize interface{}) string {
 	shape := obj.Attributes.Shape.Value
-	if strings.EqualFold(shape, d2target.ShapeCode) || strings.EqualFold(shape, d2target.ShapeClass) || strings.EqualFold(shape, d2target.ShapeSQLTable) {
+	if strings.EqualFold(shape, d2target.ShapeCode) ||
```

Comment:
Particularly, I prefer variable assignment or a function call in which, the assignment or return, have multiple lines. Tall `if` statements are quite hard to read.

The other part is, do we need `EqualFold`? If not, you could make a set (`map`) with shapes you need  to check and use that instead of chaining `||`

Reply:
i think this is very easy to read, it'd only be hard to read if they were different if statements